syntax = "proto3";

package montecarlo;

// ===== Handshake inicial del productor =====
message HandshakeRequest {
    string worker_id = 1;
}

message HandshakeReply {
    string message = 1;
    bool accepted = 2;
    int32 puntos_por_lote = 3;  // ‚Üê NECESARIO PARA TU PRODUCTOR
}

// ===== Servicio =====
service MonteCarloService {
    rpc Handshake (HandshakeRequest) returns (HandshakeReply);
}
